testgrid/cmd/transfigure/transfigure.sh

```diff
@@ -111,13 +113,21 @@ parse-args() {
 }
 
 user-from-token() {
-  user=$(curl -H "Authorization: token $(cat "${token}")" "https://api.github.com/user" 2>/dev/null | sed -n "s/\s\+\"login\": \"\(.*\)\",/\1/p")
-  echo "Using user from GitHub: ${user}"
+  if [[ -z "${user}" ]]; then
+    user=$(curl -H "Authorization: token $(cat "${token}")" "https://api.github.com/user" 2>/dev/null | sed -n "s/\s\+\"login\": \"\(.*\)\",/\1/p")
```

Comment:
Please don't do this. EIther use a real programming language, or at least take advantage of `jq` (which you can ensure exists, given you're providing this as an image).